OWML.ModHelper/ModStorage.cs

```diff
@@ -1,52 +1,26 @@
-﻿using System;
-using System.IO;
-using Newtonsoft.Json;
-using OWML.Common;
+﻿using OWML.Common;
 
 namespace OWML.ModHelper
 {
     public class ModStorage : IModStorage
     {
-        private readonly IModConsole _console;
         private readonly IModManifest _manifest;
 
-        public ModStorage(IModConsole console, IModManifest manifest)
+        public ModStorage(IModManifest manifest)
         {
-            _console = console;
             _manifest = manifest;
         }
 
         public T Load<T>(string filename)
         {
             var path = _manifest.ModFolderPath + filename;
-            if (!File.Exists(path))
-            {
-                return default;
-            }
-            try
-            {
-                var json = File.ReadAllText(path);
-                return JsonConvert.DeserializeObject<T>(json);
-            }
-            catch (Exception ex)
-            {
-                _console.WriteLine($"Error while loading {path}: {ex}");
```

Comment:
With this change, the error messages are lost

Reply:
I guess we'll still get them with verbose mode enabled (which should be the new default soon)

Reply:
You can have a better error message in JSONHelper.cs though, no? Or would that be pointless with verbose mode?

Reply:
Don't have a console, and this is (currently) static.
Could use ModConsole.Instance (obsolete/deprecated) but even that isn't always available (before manifest is loaded).

Reply:
Not sure if verbose mode is enough, couldn't think of an easy way of testing it.